src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderWidget/PaymentBuilderWidget.tsx

```diff
@@ -455,18 +453,10 @@ const PaymentBuilderWidget: FC<PaymentBuilderWidgetProps> = ({ action }) => {
           ) : (
             <>
               {finalizedAt ? (
-                <>
-                  {finalizingActions?.items[0]?.initiatorAddress ===
-                  ownerAddress ? (
-                    <FinalizeByPaymentCreatorInfo
-                      userAdddress={expenditure?.ownerAddress}
-                    />
-                  ) : (
-                    <ActionWithPermissionsInfo
-                      action={finalizingActions?.items[0]}
-                    />
-                  )}
-                </>
+                <ActionWithPermissionsInfo
+                  action={finalizingActions?.items[0]}
+                  statusText="action.executed.creator.description"
```

Comment:
This needs to show payment creator/permissions depending on the decision method used